src/stravalib/strava_model.py

```diff
@@ -669,6 +673,42 @@ class SummaryClub(MetaClub):
     Whether the club is verified or not.
     """
 
+    @field_validator("activity_types", mode="before")
```

Comment:
i wasn't sure about this. Here i added a field validator that allows us to loop through the Literal values and determine whether (sport / activity) type are valid values.